templates/base/head.tmpl

```diff
@@ -7,20 +7,30 @@
 	<title>{{if .Title}}{{.Title}} - {{end}} {{if .Repository.Name}}{{.Repository.Name}} - {{end}}{{AppName}}</title>
 	<link rel="manifest" href="{{AppSubUrl}}/manifest.json" crossorigin="use-credentials">
 
+	{{if UseServiceWorker}}
 	<script>
 		if ('serviceWorker' in navigator) {
-  			window.addEventListener('load', function() {
-    			navigator.serviceWorker.register('{{AppSubUrl}}/serviceworker.js').then(function(registration) {
-      				// Registration was successful
-      				console.log('ServiceWorker registration successful with scope: ', registration.scope);
-				}, function(err) {
-					// registration failed :(
-      				console.log('ServiceWorker registration failed: ', err);
-    			});
-  			});
+			navigator.serviceWorker.register('{{AppSubUrl}}/serviceworker.js').then(function(registration) {
+				// Registration was successful
+				console.info('ServiceWorker registration successful with scope: ', registration.scope);
+			}, function(err) {
+				// registration failed :(
+				console.info('ServiceWorker registration failed: ', err);
+			});
 		}
-
 	</script>
+	{{else}}
+	<script>
+		if ('serviceWorker' in navigator) {
+			navigator.serviceWorker.getRegistrations().then(function(registrations) {
+				for (const registration of registrations) {
```

Comment:
Even through this code will never execute in IE, won't this throw compilation error in IE?

Reply:
Indeed, I just had the same thought, fixed in latest pushh.

Reply:
lol, you fixed it seconds before my comment :+1:

Reply:
(Pretty sure it would give a SyntaxError during parsing, even if the code is never reached)